C/guide/appendixd.xml

```diff
@@ -33,15 +33,12 @@
         dialog. The first location is typically
         <filename class="directory">/usr/share/gnucash/checks</filename>, where check files
         distributed with the application can be found. The second location is the user private
-        <filename class="directory">&dir-data;checks</filename>
-        <footnote id="dir-old-chk">
+        <filename class="directory">&dir-data;checks</filename><footnote id="dir-old-chk">
           <para>Up to &app; 2.6.21 it was <filename class="directory">&dir-old;checks</filename>
           </para>
-        </footnote>
-        directory. Users may add check formats at any time (even while &app; is running) simply by
-        dropping a new <filename>*.chk</filename> file in this directory. The next time the check
-        printing dialog is opened the new check format will appear in the list of available check
-        formats.
+        </footnote>directory. Users may add check formats at any time (even while &app; is running) simply by dropping
```

Comment:
Some post processing or an improvement of our config seem required.
`</footnote> directory`
We want a space if a word follows, but none in case of a sentence sign like comma or full stop.